### PR TITLE
Add CustomTotalStatLower search term

### PR DESCRIPTION
**Pull request by Makeshift**
I added a `CustomTotalStatLower` option for my own use, building off the existing `CustomStatLower`.

I found it useful in identifying items that needed proper triage to compare with existing gear, as opposed to `CustomStatLower` being pretty much a delete-under-most-circumstances identifier.

I'm not sure if anyone except me would find it useful, but as I made the edit anyway I thought I'd open up a draft PR to discuss it. If it's considered useful, I'd be happy to clean it up. 

(inb4 this already exists, like everything else I started implementing before realising DIM already does it better than I could!)

### CODE REVIEW

**Comment by bhollis** [comment]
Would you mind explaining what this does?

**Comment by Makeshift** [comment]
> Would you mind explaining what this does?

Sure. This PR adds a `is:customtotalstatlower` term which highlight items that have a lower custom *total* stat than another item (as per the users Custom Stat Total settings) as opposed to the `customstatlower` term which highlights items where *each* of the stats within the custom stat totals are lower. 

For example, if you have a warlock custom stat total for discipline+resilience+recovery, and these two items:

This came up for me because of some confusion the `is:customstatlower` command caused me. When a user sets custom stat totals, the most immediate user-facing effect is that each armor gains a new stat that you can look at, so it becomes a fairly natural place to look when triaging items. When looking at these two items, it isn't immediately obvious why item 2 isn't highlighted:

|        | Discipline | Resilience | Recovery | Total |
|--------|------------|------------|----------|-------|
| Item 1 | 30         | 20         | 10       | 60    |
| Item 2 | 10         | 20         | 10       | 40    |

So when I used the `is:customstatlower` term, I was surprised to see that this item wasn't highlighted, being that it's worse in every custom stat total I have defined (This is the best example I could find in my now cleaned vault):
image

If you know how `customstatlower` works under the hood (is X item worse in every stat across all defined custom stats) then it makes sense, because a 23 in mobility is my best-in-type, but it wasn't super obvious to me that it was comparing all stats within the custom stat total instead of the custom stat total itself. I don't necessarily think that `customtotalstatlower` will actually get much use (realistically it's not actually very useful in identifying items that you wouldn't want in a build, since the individual stats are far more important than the total), but I also couldn't think of a nice way to modify the documentation to make `customstatlower` make a bit more sense at first glance. 

I was hoping that just the existence of this new command would act as the documentation needed to understand how `customstatlower` makes its decisions while avoiding making the user read extensive documentation to understand why something is or isn't highlighted.

**Comment by bhollis** [comment]
Unfortunately I still am not clear on what exactly this does. Let me try to rephrase it to see if I'm understanding:

> `is:customtotalstatlower` shows armor whose custom stat totals are strictly lower than another of the same type of armor. This differs from `is:customstatlower` in that it checks each custom stat *total* against other items, rather than checking the individual stats that make up each total. This is only useful if you have more than one custom stat total defined.

If I'm right, then I think I agree that this is of limited usefulness, as it requires users to have defined several custom stats, which is quite rare, and which approaches the same thing as `is:customstatlower` as you add more custom stats. I might suggest that this be closed.

I also think you may have misunderstood `is:statlower`. In the example in your table, Item 2 would be highlighted by `is:statlower`, because it is worse in at least one stat and not better in any other stat.

BTW, i18n works by editing `config/i18n.json`, see https://github.com/DestinyItemManager/DIM/blob/master/docs/CONTRIBUTING.md#development

**Comment by bhollis** [comment]
Closing, but please reopen if you feel strongly that this is something of general benefit.
